<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTransactions</class>
 <widget class="QWidget" name="ViewTransactions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="categoryLabel">
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categoryComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="subcategoryLabel">
        <property name="text">
         <string>Subcategory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="subcategoryLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Show Options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="transactionScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>386</width>
        <height>149</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="transactionTableWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
